<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKUdpClientAdvanced</class>
 <widget class="QDialog" name="SAKUdpClientAdvanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDP Client Advanced Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="9">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="10">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="enableAdvancedCheckBox">
        <property name="text">
         <string>Enable advanced settings</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="unicastCheckBox">
        <property name="text">
         <string>Unicast</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="multicastCheckBox">
        <property name="text">
         <string>Multicast</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="broadcastCheckBox">
        <property name="text">
         <string>Broadcast</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3" rowspan="2" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="exportPushButton">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="10">
    <widget class="QGroupBox" name="milticastGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Address List</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="clearUnicastPushButton">
     <property name="text">
      <string>ClearUnicast</string>
     </property>
    </widget>
   </item>
   <item row="4" column="8">
    <widget class="QPushButton" name="applyPushButton">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="clearMulticastPushButton">
     <property name="text">
      <string>ClearMulticast</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QPushButton" name="clearBroadcastPushButton">
     <property name="text">
      <string>ClearBroadcast</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="importPushButton">
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="10">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: &quot;red&quot;;
}</string>
     </property>
     <property name="text">
      <string>Afrer the parameter(s) is(are) changed! It(They) does(do) not take effect until the &quot;Apply&quot; button is clicked!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SAKUdpClientAdvanced</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
